### PR TITLE
Added: unit test;

### PR DESCRIPTION
**Pull request by NazarYanenko**

### CODE REVIEW

File: src/MandrillSenderService.php [hunk 1]
```diff
@@ -22,34 +23,17 @@ public function __construct(Mandrill $mandrill)
...
      * @param string $templateName
      * @throws CantSendException
      */
     public function sendTemplate(array $placeholders, string $templateName)
     {
-        $template = $this->mandrill->templates->info($templateName);
-        $content = str_replace(array_keys($placeholders), $placeholders, $template['code']);
-        $this->_send($placeholders['email'], $template['subject'], $content);
+        try {
+            Mail::to($placeholders['email'])->send(new MailTemplate($this->mandrill, $placeholders, $templateName));
```
Comment (sergunik) [inline src/MandrillSenderService.php]: refactor it

File: tests/ServiceTest.php [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+        return file_get_contents(__DIR__ . '/mail_template.html');
+    }
+
+    /**
+     * @return array
+     */
+    private function getPlaceholders(): array
+    {
+        return [
+            'foo' => 'bar',
```
Comment (sergunik) [inline tests/ServiceTest.php]: add valid keys

File: tests/ServiceTest.php [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    public function test_send_email()
+    {
+        Mail::fake();
+        $placeholders = $this->getPlaceholders();
+        $mandrill = $this->createMandrillMock();
+        $mandrillService = new MandrillSenderService($mandrill);
+        $mandrillService->sendTemplate($placeholders, 'tpl');
+
+        Mail::assertSent(MailTemplate::class, function ($mail) use ($placeholders) {
+            $this->assertTrue($mail->hasTo($placeholders['email']));
```
Comment (sergunik) [inline tests/ServiceTest.php]: Also check valid keys

File: tests/mail_template.html [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<html>
+<head>
+    <meta charset="utf-8"/>
+</head>
+<body>
+<h1>Email Template Example</h1>
```
Comment (sergunik) [inline tests/mail_template.html]: Add paragraphs with keys

File: composer.json [hunk 5]
```diff
@@ -20,7 +20,13 @@
     },
     "autoload": {
         "psr-4": {
-            "MandrillSender\\": "src"
+            "MandrillSender\\": "src",
+            "MailTemplate\\": "src/Mail"
```
[the remark this answers is not in the document]
Comment (NazarYanenko) [reply composer.json]: done

File: src/MandrillSenderService.php (hunk 1 above)
Comment (NazarYanenko) [reply src/MandrillSenderService.php]: done

File: tests/ServiceTest.php (hunk 3 above)
Comment (NazarYanenko) [reply tests/ServiceTest.php]: done

File: tests/mail_template.html (hunk 4 above)
Comment (NazarYanenko) [reply tests/mail_template.html]: done

**Pull request merged**